src/Workspaces/Core/Portable/Serialization/SerializableSourceText.cs

```diff
@@ -28,7 +30,7 @@ internal sealed class SerializableSourceText
     /// <remarks>
     /// Exactly one of <see cref="_storage"/> or <see cref="_text"/> will be non-<see langword="null"/>.
     /// </remarks>
-    private readonly ITemporaryTextStorageWithName? _storage;
+    private readonly TemporaryTextStorage? _storage;
```

Comment:
moved to stronger types, as several of these interfaces only have a single impl ever.